bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -107,6 +110,7 @@ CGCON-BSTC01-00017,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and
 CGCON-BSTC01-00018,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0018,,,,,Woods Hole Oceanographic Institution
 CGCON-BSTC01-00040,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0040,,,,,Woods Hole Oceanographic Institution
 CGCON-BSTC01-00042,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0042,,,,,Woods Hole Oceanographic Institution
+CGCON-BSTC01-00043,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,3701-00143-00000-00043,,,,,Woods Hole Oceanographic Institution
```

Comment:
The S/N should be 3701-00143-00001-00043  (it's missing the 1 in the third set of numbers).